### ISSUE TITLE
DMs not sending

### ISSUE THREAD

**Issue by dorry-kun** [cezarsa/silver_bird#421]
For the past few days, I haven't been able to send DMs. Same thing is happening to my girlfriend on a different OS. It works fine through web Twitter.
Does anybody have the same problem?
I suspect maybe something has changed in chrome lately, that could be causing it. It did start at about the same time I last upgraded it.

**Comment by jcmeloni-zz** [comment]
I have the same issue, also at the same time as an upgrade.  I don't believe Twitter changed anything in the API, but I could be wrong...

**Comment by Wilsonx1999** [comment]
I've been having the same issue for at least a week, not a deal breaker, but kind of annoying.

**Comment by thebeercard** [comment]
I am seeing the same problem. I am on the beta channel of Chrome running on Windows 8.1.

**Comment by jcmeloni-zz** [comment]
FWIW, I can send DMs again, and the only change to my system was a Windows update -- not even a Chrome update.  So, who knows?

**Comment by thebeercard** [comment]
Working for me too.

**Comment by Wilsonx1999** [comment]
Yeah, it just started working here too. Have no idea if there was any other change, but I'm not complaining.

**Comment by dorry-kun** [comment]
I can also send DMs now. No system updates, but I upgraded chrome yesterday again (from 33 to 34)...

**Comment by FernandoMiguel** [comment]
chrome dev here ... cant send DMs from Win8 or Ubuntu :(
